Formula/s/swiftly.rb

```diff
@@ -0,0 +1,36 @@
+class Swiftly < Formula
+  desc "Swift toolchain installer and manager"
+  homepage "https://github.com/swiftlang/swiftly"
+  url "https://github.com/swiftlang/swiftly.git",
+      tag:      "1.0.0",
+      revision: "a9eecca341e6d5047c744a165bfe5bbf239987f5"
+  license "Apache-2.0"
+  head "https://github.com/swiftlang/swiftly.git", branch: "main"
+
+  # On Linux, SPM can't find zlib installed by brew.
+  depends_on :macos
+
+  depends_on macos: :ventura
+  depends_on xcode: "8.0"
+
+  uses_from_macos "swift" => :build, since: :sonoma # swift 5.10+
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "libarchive"
+  end
+
+  def install
+    args = if OS.mac?
+      ["--disable-sandbox"]
+    else
+      %w[--static-swift-stdlib -Xcc -I#{HOMEBREW_PREFIX}/include -Xlinker -L#{HOMEBREW_PREFIX}/lib]
+    end
+    system "swift", "build", *args, "--configuration", "release", "--product", "swiftly"
```

Comment:
```suggestion
    system "swift", "build", *args, "--configuration", "release", "--product", "swiftly", "--disable-sandbox"
```
if you're only supporting macOS